minari/serialization.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import json
+from typing import Dict, Union
+
+import gymnasium as gym
+import numpy as np
+
+
+def serialize_space(space: gym.spaces.Space, to_string=True) -> Union[Dict, str]:
```

Comment:
a `if` branch for every space is the only thing we can do now
But I am wondering if it would be possible (and clean for Gymnasium) to record the initialization argument of spaces as done for Wrappers @pseudo-rnd-thoughts